xml/installation-installation-configure-deployer-repos.xml

```diff
@@ -0,0 +1,499 @@
+    updates for all software need to be available on or accessible from the
+    &admserv;. In this configuration step, these repositories are made
+    available. There are two types of repositories:
+   </para>
+   <para>
+    <emphasis role="bold">Product Media Repositories</emphasis>: Product media
+    repositories are copies of the installation media. They need to be
+    directly copied to the &lcm; server, <quote>loop-mounted</quote> from an iso
+    image, or mounted from a remote server via NFS. Affected are &cloudos; and
+    &productname; &productnumber;. These are static repositories; they do not change or receive updates. See <xref linkend="sec.depl.adm_conf.repos.product"/> for setup
+    instructions.
+   </para>
+   <para>
+    <emphasis role="bold">Update and Pool Repositories</emphasis>: Update and
+    Pool repositories are provided by the &scc;. They contain all updates and
+    patches for the products and extensions. To make them available for
+    &cloud; they need to be mirrored from the &scc;. Since their content is
+    regularly updated, they must be kept in synchronization with &scc;. For
+    these purposes, &suse; provides either the &smtool; (&smt;) or the
+    &susemgr;.
+   </para>
+  </abstract>
+ </info>
+ <sect1 xml:id="sec.depl.adm_conf.repos.product">
+  <title>Copying the Product Media Repositories</title>
+  <para>
+   The files in the product repositories for &sls; and &cloud; do not
+   change, therefore they do not need to be synchronized with a remote
+   source. If you have installed the product media from a downloaded ISO image,
+   the product repositories will automatically be made available to the client
+   nodes and these steps can be skipped. Similarly, if you have configured the
+   &lcm; node as an SMT mirror, the product repositories will automatically be
+   made available to the client nodes and these steps can be skipped.
+   Otherwise, it is sufficient to either copy the data (from a remote host or
+   the installation media), to mount the product repository from a remote
+   server via <literal>NFS</literal>, or to loop mount a copy of the
+   installation images.
+   </para>
+   <para>
+    The following product media must be copied to the specified
+    directories:
+   </para>
+   <table>
+    <title>Local Product Repositories for &cloud;</title>
+    <tgroup cols="2">
+     <colspec colnum="1" colname="1" colwidth="40*"/>
+     <colspec colnum="2" colname="2" colwidth="60*"/>
+     <thead>
+      <row>
+       <entry>
+        <para>
+         Repository
+        </para>
+       </entry>
+       <entry>
+        <para>
+         Directory
+        </para>
+       </entry>
+      </row>
+     </thead>
+     <tbody>
+      <row>
+       <entry>
+        <para>
+         &cloudos; DVD #1
+        </para>
+       </entry>
+       <entry>
+        <para>
+         <filename>/srv/www/suse-12.3/x86_64/repos/SLES12-SP3-Pool</filename>
```

Comment:
This could also use `&www_dir;` I guess.

But isn't  `SLES12-SP3-Pool` supposed to be used for the -Pool repo synched via SMT/Manager?

Reply:
I copied this from http://docserv.nue.suse.com/documents/Cloud7/suse-openstack-cloud-deployment/html/cha.depl.repo_conf.html#sec.depl.adm_conf.repos.product

I guess this might only be important for how crowbar does provisioning and not relevant here, will remove it

Reply:
Done

Reply:
So yes this was how crowbar did provisioning, I guess the equivalent for Ardana would be this: http://docserv.nue.suse.com/documents/SUSE_OpenStack_Cloud_8/hpe-helion-openstack-cloud-installation/single-html/#idm140509424462576

We don't have a converged approach for that yet, so I'll only mention the Cloud product media here and not the SLES product media.